### PR TITLE
docs(js): executors examples

### PR DESCRIPTION
**Pull request by nartc**
## Related Issue(s)

Fixes #

### CODE REVIEW

**Review by isaacplmann — APPROVED** [review]

File: packages/js/docs/swc-examples.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+## Examples
+
+{% tabs %}
+{% tab label="Inline libraries" %}
```
Comment (bcabanes) [inline packages/js/docs/swc-examples.md:4]: I see you are using tabs here but only one is used, is there a second one coming?

File: packages/js/docs/tsc-examples.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
+## Examples
+
+{% tabs %}
+{% tab label="Using TypeScript Transformer Plugins" %}
+
+`@nrwl/js:tsc` can run the [TypeScript Transformers](https://github.com/madou/typescript-transformer-handbook) by using the `transformers` option.
+
+```json
```
Comment (bcabanes) [inline packages/js/docs/tsc-examples.md:8]: If needed you can pass the name of the file in a code fence with the `{% fileName="path/to/my/file.ext" %}`.
More details here https://github.com/nrwl/nx/blob/master/docs/README.md#code.

File: packages/js/docs/swc-examples.md (hunk 1 above)
Comment (nartc) [reply packages/js/docs/swc-examples.md:4]: copy-pasta. Added another one.

File: packages/js/docs/tsc-examples.md (hunk 2 above)
Comment (nartc) [reply packages/js/docs/tsc-examples.md]: Thanks!

**Pull request merged**